nf_core/modules/modules_json.py

```diff
@@ -469,6 +480,9 @@ def check_up_to_date(self):
 
         If a module/subworkflow is installed but the entry in 'modules.json' is missing we iterate through
         the commit log in the remote to try to determine the SHA.
+
+        Check that we have the "installed" value in 'modules.json', otherwise add it.
+        Assume that the modules/subworkflows were installed by and nf-core command (don't track installed by subworkflows).
```

Comment:
```suggestion
        Assume that the modules/subworkflows were installed by an nf-core command (don't track installed by subworkflows).
```
I don't quite understand the "don't track installed by subworkflows". Do you mean "don't track subworkflows installed by subworkflows"?